bundles/org.smarthomej.binding.viessmann/src/main/java/org/smarthomej/binding/viessmann/internal/ViessmannDynamicCommandDescriptionProvider.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2021-2022 Contributors to the SmartHome/J project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.smarthomej.binding.viessmann.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.events.EventPublisher;
+import org.openhab.core.thing.binding.BaseDynamicCommandDescriptionProvider;
+import org.openhab.core.thing.i18n.ChannelTypeI18nLocalizationService;
+import org.openhab.core.thing.link.ItemChannelLinkRegistry;
+import org.openhab.core.thing.type.DynamicCommandDescriptionProvider;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * Dynamic provider of command options.
+ *
+ * @author Ronny Grun - Initial contribution
+ */
+@Component(service = { DynamicCommandDescriptionProvider.class, ViessmannDynamicCommandDescriptionProvider.class })
+@NonNullByDefault
+public class ViessmannDynamicCommandDescriptionProvider extends BaseDynamicCommandDescriptionProvider {
```

Comment:
Please correct me if I am wrong: you can remove the entire class.

Reply:
yes of course :) I'll remove this class